app/src/pages/homes.tsx

```diff
@@ -0,0 +1,69 @@
+import type { NextPage } from "next";
+import Head from "next/head";
+import React from "react";
+import Image from "next/image";
+import { trpc } from "utils/trpc";
+import Navbar from "@components/navbar";
+import { useState } from 'react';
+
+
+const HomesPage: NextPage = () => {
```

Comment:
The styling on this page doesn't seem complete to me. I think going forward we need to have more of the styles finished when merging things in. Things like displaying homes seems like they should be in organized cards with the image displayed imo. The footer seems to close as well to the content of the page and the logo and welcome seem out of place/take up to much room.

Reply:
Correct, the home page is more of a temporary page as well until we have the navbar completed. Remember from the figma mocks we are planning on having home selection to be selected from the center of the navbar. This page will become a fallback for when there are no homes the user is apart of